pkg/platform/runtime/setup/setup.go

```diff
@@ -482,6 +463,7 @@ func (s *Setup) fetchAndInstallArtifactsFromBuildPlan(installFunc artifactInstal
 		s.analytics.Event(anaConsts.CatRuntimeDebug, anaConsts.ActRuntimeBuild, dimensions)
 	}
 
+	includeBuildtimeClosure := strings.EqualFold(os.Getenv(constants.InstallBuildDependencies), "true") || !buildResult.BuildReady
```

Comment:
I understand we would want to report progress on build time artifacts if the build is not ready. But I can't tell from reading this whether this is saying "include it for progress reporting" or "include it for everything". Worth renaming the var to make it explicit.